Symfony/CS/Tests/Fixer/AbstractFixerTestBase.php

```diff
@@ -70,13 +70,13 @@ protected function makeTest($expected, $input = null, \SplFileInfo $file = null,
             $tokens = Tokens::fromCode($fixedCode); // Load cached collection (used by the fixer)
             Tokens::clearCache();
             $expectedTokens = Tokens::fromCode($fixedCode); // Load the expected collection based on PHP parsing
-            $this->assertTokens($expectedTokens, $tokens);
+            self::assertTokens($expectedTokens, $tokens);
         }
 
         $this->assertSame($expected, $fileIsSupported ? $fixer->fix($file, $expected) : $expected);
     }
 
-    private function assertTokens(Tokens $expectedTokens, Tokens $tokens)
+    public static function assertTokens(Tokens $expectedTokens, Tokens $tokens)
```

Comment:
I would like to use this assert in the `TokensTest`, it feels odd to make that one extend the `AbstractFixerTestBase`